releasenotes/notes/detect-attacks-on-the-server.request.cookies-c138bee409617ff0.yaml

```diff
@@ -0,0 +1,5 @@
+---
+features:
+  - |
+    Detect attacks on the server.request.cookies
```

Comment:
We should probably prefix this with `ASM: ` so people know this only relates to that specific product.

Would ASM customers know what `server.request.cookies` means? or should we be a little more verbose?

> ASM: Detect attacks on Flask request cookies

what do you think?

Reply:
You're right, I will update the release notes with a message more verbose :+1: